.github/workflows/build-and-quality-checks-v3.yml

```diff
@@ -2,7 +2,7 @@ name: Build & Code Quality Checks v3
 
 on:
   pull_request:
-    branches: ['v3-production', 'v3-staging']
+    branches: ['v3-staging', 'develop', 'master']
```

Comment:
Can we please use `main` instead of `master`? It's more appropriate.

Reply:
sure will do this

Reply:
done